meiki_ui/src/components/explorer/Panel.svelte

```diff
@@ -8,7 +8,7 @@
     class=" bg-gray-50 border-gray-200 border-r flex flex-col gap-1 py-5"
     style="width: {width}"
     data-cy="explorer"
-    transition:fly={{ x: -400, duration: 350, opacity: 1 }}
+    transition:fly|local={{ x: -400, duration: 120, opacity: 1 }}
```

Comment:
Why local here?

Reply:
Because when we logout (transition to another page), we will have to wait for the panel to close, local will prevent that.

Ref: https://github.com/sveltejs/kit/issues/628#issuecomment-547246182